### PR TITLE
deno-unboxing

### PR DESCRIPTION
**Pull request by Akashmallareddy**

### CODE REVIEW

File: _posts/2020-05-20-deno-unboxing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Deno 1.0"
+author: Akash Rao Mallareddy
```
Comment (SriSatyaLokesh) [inline _posts/2020-05-20-deno-unboxing.md:4]: You need to keep the author name as contributor. We are going to create a speical page for contributors
At the end you can describe your self with your own portfolio URL.

File: _posts/2020-05-20-deno-unboxing.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title:  "Deno 1.0"
+author: Akash Rao Mallareddy
+categories: [ web framework ]
+image: assets/images/deno.png
+tags: [deno, node.js alternative]
+---
+
+**A Brief Introduction To Deno**
```
Comment (SriSatyaLokesh) [inline _posts/2020-05-20-deno-unboxing.md:10]: All of these should be #### means h4.
Bold is not enough.

File: _posts/2020-05-20-deno-unboxing.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+There are some exciting new features in Deno. They are:
+
+**1. Security is integrated**
+
+![Amazon App Store]({{ site.baseurl }}/assets/images/alternatives/google-play-store/amazon-app-store.png)
+
+Node.js allows you to access read and write into the filesystem, access environment variables, make outgoing requests, etc. As well as it also opens up a security risk if you are not careful when writing code.
+
+So instead, Deno uses command-line arguments to enable or disable access to different security features. So if you need your script to be able to access the /etc folder, you can do:
+#####
```
Comment (SriSatyaLokesh) [inline _posts/2020-05-20-deno-unboxing.md:26]: this empty h5 is completely irrelevant. Please remove this.

File: _posts/2020-05-20-deno-unboxing.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+**2. Complete standard library**
+
+Deno claims to have a complete standard library allowing developers to use official tools to perform basic tasks and only requiring the use of external libraries for complex tasks and it comes with the tools to add color to terminal text, generate UUIDs, work with external data structures,  and for writing WebSockets.
+
+**3. Integrated Typescript**
+
+In Deno, by default the integration into JavaScript is done internally, so no need for external tooling. By default Deno takes care of a lot, you can overwrite the configuration by using your own tsconfig.json file:
+#####
+```
```
Comment (SriSatyaLokesh) [inline _posts/2020-05-20-deno-unboxing.md:40]: make it \```sh

File: _posts/2020-05-20-deno-unboxing.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+##### Using Scoop (Windows):
+```
+$ scoop install deno
+```
+
+
+**Where to learn Deno?**
+
+https://www.freecodecamp.org/news/the-deno-handbook/
```
Comment (SriSatyaLokesh) [inline _posts/2020-05-20-deno-unboxing.md:78]: You'd have provided official docs

**Comment by Akashmallareddy** [comment]
Updated

File: _posts/2020-05-20-deno-unboxing.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+```
+The default configuration of Deno is to use strict mode, so any ill-advised coding practice will be alerted immediately.
+
+
+#### Deno Installation
+
+Deno ships as a single executable with no dependencies. You can install it using the installers below: 
+
+##### Using Shell (macOS, Linux):
+```
```
Comment (puneethkanna) [inline _posts/2020-05-20-deno-unboxing.md:49]: Make the language to sh

File: _posts/2020-05-20-deno-unboxing.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+$ cargo install deno
+```
+##### Using Homebrew (macOS):
+```
+$ brew install deno
+```
+##### Using Chocolatey (Windows):
+```
+$ choco install deno
+```
```
Comment (puneethkanna) [inline _posts/2020-05-20-deno-unboxing.md:67]: Change the language to appropriate language.

File: _posts/2020-05-20-deno-unboxing.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+##### Using Scoop (Windows):
+```
+$ scoop install deno
+```
```
Comment (puneethkanna) [inline _posts/2020-05-20-deno-unboxing.md:71]: Change the language to appropriate language.

File: _posts/2020-05-20-deno-unboxing.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+---
+
+#### A Brief Introduction To Deno 1.0
+
+Deno is the creation of Ryan Dahl, which is a new JavaScript runtime for the backend, but instead of being written in C++, it’s written in Rust, based on the Tokio platform (which provides the asynchronous runtime needed by JavaScript), running on Google’s V8 engine. It is also a new runtime for executing JavaScript and TypeScript outside of the web browser. 
+
+
+#### Why should we use this?   
+
+There are some exciting new features in Deno. They are:
```
Comment (puneethkanna) [inline _posts/2020-05-20-deno-unboxing.md:17]: Replace by:
Generally we may get a doubt, why should I use it, here are some of it's features.

File: _posts/2020-05-20-deno-unboxing.md (hunk 6 above)
Comment (puneethkanna) [reply _posts/2020-05-20-deno-unboxing.md:49]:
```bash
THE CODE
```

File: _posts/2020-05-20-deno-unboxing.md (hunk 9 above)
Comment (SriSatyaLokesh) [reply _posts/2020-05-20-deno-unboxing.md:17]: I will fix all those @puneethkanna

Comment (puneethkanna) [reply _posts/2020-05-20-deno-unboxing.md:17]: Okay @SriSatyaLokesh bro

**Pull request merged**